gptcache/manager/object_data/manager.py

```diff
@@ -0,0 +1,25 @@
+from gptcache.utils.error import NotFoundStoreError
+
+
+class ObjectBase:
+    """
+    ObjectBase to manager the object storage.
+    """
+
+    def __init__(self):
+        raise EnvironmentError(
+            "CacheBase is designed to be instantiated, please using the `CacheBase.get(name)`."
+        )
+
+    @staticmethod
+    def get(name, **kwargs):
+        if name == "local":
+            from gptcache.manager.object_data.local_storage import LocalObjectStorage  # pylint: disable=import-outside-toplevel
+            object_base = LocalObjectStorage(kwargs.get("path", "./"))
```

Comment:
the default value `./local_obj` may be more suitable

Reply:
fixed